### DISCUSSION TITLE
Major Update Now Available!

### DISCUSSION THREAD

**Discussion by sircharlo** [sircharlo/meeting-media-manager#2753] [New releases]
🎉 **We’re thrilled to announce the official release of the new version of Meeting Media Manager (M³)!** 🎉

After **months of testing** and invaluable feedback from all of you, the fully redesigned M³ is now available for everyone! 🙌

This version brings everything we’ve worked hard on together—an **intuitive drag-and-drop interface** and **simplified media management**, making your tasks as an audio-video attendant **easier than ever**. We couldn't have made this happen without your help in the design, app review, and feedback! 💪

A **huge thank you** to everyone who tested, shared feedback, and helped shape this new release. 💖 Your input ensured that this version is stable, reliable, and packed with everything you need.

🚀 **What’s next?**
- Please continue to report any issues you encounter.
- Feel free to open new discussions for **feature requests** or **enhancements**. We're always looking for ways to improve!

We’re excited for you to start using the new M³ for your meetings. 🎥💻 We hope it brings a smoother and more enjoyable experience! 😊

**Comment by dagandrew** [comment]
Hi, thank you very much for the new update. Interface is so amazing! Seeing what song playing in the background is so nice! 

I was really excited to try drag-and-drop functionality, but it didn't work for me on Windows 11 21H2, although I can drag-and-drop on other apps. Could it be fixed? 

![image](https://github.com/user-attachments/assets/32c48c1b-8f0b-40aa-b634-63481a013a1b)

And I have a suggestion: can you paste (by just pressing ctrl + v) the images or videos that you've copied into clipboard (for example, you've copied the image from the browser and you could just paste it too when you are on the main window?

Another suggestion is move the badges with paragraphs and songs to the thumbnail (with a background, so it could be visible)
![Untitled](https://github.com/user-attachments/assets/2957a53e-59cf-4588-ab30-c318e162a7e3)

Also I've noticed that the background image I've set is reset every time I close the app. 

And could make Media Player Window remember its last set window size? 

And of course I'm waiting for more keyboard shortcuts like going to next media. 

Also if you know that particular video or picture won't be shown, can you just hide from the playlist? (you don't have to put the trash icon next to every video, but maybe right click and hide or detete?)

Anyway, these are just minor complaints. Thank you so much for your work!

**Reply by e-Feitosa** [reply]
>I was really excited to try drag-and-drop functionality, but it didn't work for me on Windows 11 21H2, although I can drag-and-drop on other apps.

**Português:** Eu também tive esse mesmo problema com arrastar e soltar.
**Inglês:** I also had the same problem with drag-and-drop.

https://github.com/sircharlo/mmm-refactor/discussions/2#discussioncomment-10977428

**Reply by sircharlo** [reply]
Thanks for your nice comment! @dagandrew 
Can you open a new issue for every problem in the official repo please? This will help me to categorize these things 😄
Also, one new discussion for every feature request. 

Thank you!

**Reply by dagandrew** [reply]
Of course, thanks

**Comment by Danreynolds124** [comment]
Hello, on the Mac version when sharing on zoom if the media display window is full screen on the second display zoom does not pick it up as an available app window to be shared. We can only share via zoom with it in windowed mode. Any fix?

**Reply by sircharlo** [reply]
Thanks for your message! Can you open an issue for this please?

**Comment by Danreynolds124** [comment]
How do I do that? I’ve never used GitHub before 😁


__

Dan Reynolds

815-416-9181
585-721-9181 (cell)

www.danreynoldsphotography.com

**Comment by matuza83** [comment]
In the last update, the previous video that showed a countdown timer that started automatically, could it be placed as an option to be activated or deactivated as it happens with the JW emblem on the black screen that you can activate or remove if you want?

**Reply by e-Feitosa** [reply]
I believe you are referring to this feature that had no previous version before this new version. No doubt it was an excellent resource that made it much easier to reproduce the first media on the list, I used here too. My request to have this option back as part of the M3 advanced settings is still valid.

![image](https://github.com/user-attachments/assets/77a0cfcc-f54d-4df9-bc92-dad0a372f7c2)

**Reply by sircharlo** [reply]
> In the last update, the previous video that showed a countdown timer that started automatically, could it be placed as an option to be activated or deactivated

In line with direction about what should and should not be displayed on the KH screens before the opening song, this feature will not be implemented.

> as it happens with the JW emblem on the black screen that you can activate or remove if you want?

Similarly, in line with direction about what should and should not be displayed on the KH screens before the opening song, after the concluding prayer, and during the meeting, this feature will not be implemented. 


Thank you for your understanding!

**Reply by e-Feitosa** [reply]
> Automatically play first media item

@sircharlo, Could this feature be added for special meetings (other meetings)?

**Reply by sircharlo** [reply]
> @sircharlo, Could this feature be added for special meetings (other meetings)?

What scenarios would this be used in?

**Reply by fschein** [reply]
Dear @sircharlo,

I completely understand that this question has likely been asked numerous times and that you're probably tired of addressing it. I sincerely appreciate your patience and the clarity you've provided so far.

After reviewing the documents you referenced, we couldn't find a specific restriction regarding the automatic playback of a video before the meeting begins. In our case, we were hoping to have the option to play a video with a countdown timer before the start of the meeting, similar to the previous feature that allowed for automatic playback of the first media item in the list. Additionally, we were hoping for the possibility of having special media for special meetings, which would be helpful in certain situations.

Given that the documents primarily focus on media usage during the meeting itself, we wanted to respectfully inquire again if these features could be considered, as we didn't find anything in the documents that explicitly prohibits them.

We understand that there are guidelines to follow, and we deeply respect that. Our intention is simply to seek further clarification regarding these features and whether they might be allowed under the existing guidelines.

Thank you again for your time and understanding.

**Reply by sircharlo** [reply]
@fschein :

Thank you for your thoughtful question and for taking the time to review the documents. Please know that I’m not tired of addressing these inquiries—it’s always a pleasure to provide clarification or the reason for design decisions when needed.

Regarding your question, the direction found at _sfl_ 21:38(2) is the main reason this feature will not be implemented. This subpoint directs that the digital yeartext should be displayed before and after meetings, and that no additional artwork should be displayed or combined with the digital yeartext.

Combine this with the S-141, referenced by _sfl_ 20:23, which states that we should not use any moving video footage unless directed to do so by the organisation. 

I hope this helps clarify things. Thank you again for your understanding as to why we would feel uncomfortable developing this feature, and your respectful approach to this matter.

Warm regards

**Comment by e-Feitosa** [comment]
> What scenarios would this be used in?

The idea was more to use it for special events. Here, for example, the Kingdom Hall is used for circuit pioneer schools, annual meetings with pioneers, etc. It was more for these events that are not regular meetings. Here, for example: Every month the congregation watches the "JWB Broadcasting" together at the Kingdom Hall (local arrangement). It was only for occasions of this nature, and that's why I asked if this feature would be available in the additional meetings option.

**Reply by sircharlo** [reply]
Right; and how would a "play first media" option be of use in a scenario like that?

**Reply by mtdvlpr** [reply]
I don't really see how useful this would be. In essence, we would be automating a single click action. M3 already starts in the present media page. This would save maybe 5 seconds at best?

**Comment by matuza83** [comment]
TEMPO 2 MINUTOS
<https://drive.google.com/file/d/1fqGEg_8wur6qLmJ3CefYfEeooZC6gckT/view?usp=drivesdk>


Montagem de 2 vídeos jw - última reunião stream no Brasil finalizando c/
trecho das mídias do site jw.org

Poderiam criar a opção [ATIVAR ou NÃO ATIVAR] no programa, e deixar p/ os
irmãos que vão usar o programa a decisão a respeito.

Lá na onde fazemos uso, colocamos manualmente após a última atualização.

Att.
    Matuza

Em sex., 29 de nov. de 2024 05:12, Manoah Tervoort ***@***.***>
escreveu:

**Reply by sircharlo** [reply]
Thanks for you example; I think we'll leave things as is. Operators can play the video by clicking on the Play button when needed. Adding anything more for this would be an edge case that wouldn't bring much benefit, in my opinion.
